test/AggregationTallyScript.spec.ts

```diff
@@ -183,6 +183,41 @@ describe('AggregationTallyScript', () => {
     })
   })
 
+  describe('deleteOperator', () => {
+    it('without repeated filters', () => {
+      const mirScript: MirAggregationTallyScript = {
+        filters: [
+          [AggregationTallyFilter.deviationAbsolute, 3],
+          AggregationTallyFilter.mode
+        ],
+        reducer: 0x02,
+      }
+      const cache = new Cache()
+      const script = new AggregationTallyScript(cache, mirScript)
+
+      script.deleteOperator(2)
```

Comment:
What is this 2?

Reply:
The id that is assigned to the operator I want to delete. I change it to make it explicit